src/main/java/com/amazon/opendistroforelasticsearch/knn/plugin/script/KNNScoringUtil.java

```diff
@@ -1,9 +1,24 @@
+/*
+ *   Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License").
+ *   You may not use this file except in compliance with the License.
+ *   A copy of the License is located at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   or in the "license" file accompanying this file. This file is distributed
+ *   on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ *   express or implied. See the License for the specific language governing
+ *   permissions and limitations under the License.
+ */
+
 package com.amazon.opendistroforelasticsearch.knn.plugin.script;
 
 import org.apache.logging.log4j.LogManager;
 import org.apache.logging.log4j.Logger;
 
-import java.util.ArrayList;
+import java.math.BigInteger;
 
 public class KNNScoringUtil {
```

Comment:
Should we merge this class with KNNScoringSpaceUtil?

Reply:
So, I thought it might be worth keeping KNNScoringSpaceUtil and KNNScoringUtil separate. KNNScoringSpaceUtil contains utility functions for parsing/preparing queries, KNNScoringUtil contains the methods that compute the distance for a particular space.

I could rename KNNScoringUtil to KNNDistanceUtil if that sounds better? What do you think @vamshin @VijayanB ?